src/main/java/kr/codesqaud/cafe/repository/h2Repository/H2ArticleRepository.java

```diff
@@ -25,19 +25,26 @@ public H2ArticleRepository(DataSource dataSource) {
         jdbcTemplate = new JdbcTemplate(dataSource);
     }
 
-    public void save(Article article) {
-        String sql = "insert into articles(writer, title, contents, timeStamp) values (?, ?, ?, ?)";
+    public int save(Article article) {
+        String sql = "insert into articles(writer, title, contents) values (?, ?, ?)";
 
-        jdbcTemplate.update(sql, article.getWriter(), article.getTitle(), article.getContents(), article.getTimeStamp());
+        return jdbcTemplate.update(sql, article.getWriter(), article.getTitle(), article.getContents());
     }
 
-    public int delete(int index) {
+    @Override
+    public int update(int articleId, String title, String content) {
+        String sql = "update articles set title = ?, contents = ? where articleId = ?";
+
+        return jdbcTemplate.update(sql, title, content ,articleId);
+    }
+
+    public int delete(int articleId) {
         String sql = "delete from articles where articleId=?";
 
-        return jdbcTemplate.update(sql, index);
+        return jdbcTemplate.update(sql, articleId);
     }
 
-    public Article findByIndex(int index) {
+    public Article findByArticleId(int index) {
```

Comment:
```suggestion
    public Optional<Article> findByArticleId(int index) {
```
결과가 없을때 null보다는 Optional이겠죠?